apps/andi/lib/andi/input_schemas/datasets.ex

```diff
@@ -135,6 +135,12 @@ defmodule Andi.InputSchemas.Datasets do
     update(from_dataset, %{ingestedTime: iso_ingested_time})
   end
 
+  def update_submission_status(dataset_id, status) do
+    from_dataset = get(dataset_id)
+
+    update(from_dataset, %{submission_status: status})
```

Comment:
Some things like this could be optimized to not even do a `get` since all they need is the dataset id. However, it's not killing us AFAIK